contracts/OffsetHelper.sol

```diff
@@ -108,15 +83,75 @@ contract OffsetHelper is OffsetHelperStorage {
     }
 
     modifier onlySwappable(address _token) {
-        require(isSwappable(_token), "Token not swappable");
+        require(isSwappable(_token), "Path doesn't yet exists.");
+
+        _;
+    }
+
+    modifier nativeTokenChain() {
```

Comment:
If its checking only for Celo, maybe the naming of this modifier should be celo specific?

Reply:
yes. I think it's Ubeswap specific/ or Celo and I haven't seen this with any other chain yet.